src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -70,12 +79,23 @@ public Command updateInternalEncoder() {
         return runOnce(io::updateInternalEncoder).withName("Update hood internal encoder");
     }
 
+    private Pose3d getPose3d(double angle) {
+        return new Pose3d(
+                HoodConstants.ROOT_POSITION.getX(),
+                0.0,
+                HoodConstants.ROOT_POSITION.getY(),
+                new Rotation3d(0.0, -angle, 0.0));
+    }
+
     /** Updates the state of the hood. */
     @Override
     public void periodic() {
         io.updateInputs();
         Logger.processInputs("Hood", inputs);
 
         hood.setAngle(inputs.angle.in(Units.Degrees));
+
+        Logger.recordOutput("HoodPose", getPose3d(inputs.angle.in(Units.Degrees)));
+        SmartDashboard.putData("HoodMech", mechanism2d);
```

Comment:
The mechanism should instead be `@AutoLogOutput`.

Reply:
that was already the case, just removed the smartDahsboard.putData
a2f3916